src/Main.java

```diff
@@ -1,39 +1,40 @@
 import java.time.LocalDate;
+
 public class Main {
     public static void determineTheYear(int year) {
-        if (year >= 1584 && (year % 4) == 0) {
-            System.out.println(year + " год является високосным");
-        } else if (year >= 1584 && (year % 400) == 0) {
+        if (year >= 1584 && (year % 4) == 0 || year >= 1584 && (year % 400) == 0) {
```

Comment:
условие високосного года неправильное. Правильное условие:
```suggestion
        if (year >= 1584 && year % 4 == 0 && year % 100 != 0 || year % 400 == 0) {
```